tock/tock/context_processors.py

```diff
@@ -0,0 +1,13 @@
+def version_url(request):
+    import re
+    from django.conf import settings
```

Comment:
It's not a big deal, but is there a reason the imports are here rather than up top as PEP-8 would usually have them?